tidb-on-kubernetes/maintain-node.md

```diff
@@ -0,0 +1,163 @@
+---
```

Comment:
In https://github.com/pingcap/docs-cn/pull/1419, the directory name is `tidb-in-kubernetes`. Please keep it consistent. Use `tidb-in-kubernetes`.

Reply:
good catch